modules/create_api_gateway_keys/main.py

```diff
@@ -106,14 +110,15 @@ def main(args, pacu_main):
             api_gateway_data[region] = api_keys[region]
     session.update(pacu_main.database, APIGateway=api_gateway_data)
 
+    print('\n{} completed.\n'.format(module_info['name']))
     return summary_data
 
 
 def summary(data, pacu_main):
     out = ''
     if data.get('cleanup'):
-        out += '  Old keys were removed.\n'
-    else:
-        out += '  Old keys were not removed.\n'
-    out += '  {} key(s) were created.\n'.format(data['keys_created'])
+        out += '  Old keys removed.\n'
```

Comment:
It could, I guess the only reason is you can cleanup your keys and then generate a new set. How should I go into more detail about how the keys are stored in the database? Tell them to look in APIGateway column for the current session?